src/BenchmarkDotNet/Running/BenchmarkConverter.cs

```diff
@@ -155,18 +159,25 @@ private static MethodInfo GetTargetedMatchingMethod(MethodInfo benchmarkMethod,
             MethodInfo iterationSetupMethod,
             MethodInfo iterationCleanupMethod,
             BenchmarkAttribute attr,
+            BenchmarkDescriptionAttribute? classDescription,
+            BenchmarkDescriptionAttribute? methodDescription,
             MethodInfo[] targetMethods,
             IConfig config)
         {
             var categoryDiscoverer = config.CategoryDiscoverer ?? DefaultCategoryDiscoverer.Instance;
+            string description = attr?.Description;
+                if (description is null)
+                    description = methodDescription?.Description;
+                if (description is null)
+                    description = classDescription?.Description;
```

Comment:
I'm not sure if this makes sense. We want to override the `Type` description, not the method description.